### PR TITLE
CC - Add Schema documentation

### PR DESCRIPTION
**Pull request by nMoncho**

### CODE REVIEW

File: core-curriculum/schema/knowledge-graph.json [hunk 1]
```diff
@@ -4,39 +4,62 @@
   "id": "lunatech-core-curriculum-knowledge-graph-schema",
   "properties": {
     "description": {
+      "description": "Provides a description of the subject, how does it fit in the Software Development environment, can be a summary or short introduction.",
+      "id": "/properties/description",
+      "maxLength": 500,
       "type": "string"
     },
     "id": {
+      "description": "An unique identifier for this subject, allowing to refer the subject by it.",
```
Comment (eamelink) [inline core-curriculum/schema/knowledge-graph.json]: A unique

File: core-curriculum/schema/knowledge-graph.json [hunk 2]
```diff
@@ -90,6 +141,8 @@
           },
           "tags": {
             "items": {
+              "description": "A list of tag for categorising the topic.",
```
Comment (eamelink) [inline core-curriculum/schema/knowledge-graph.json]: You spelled categorizing with a 'z' elsewhere :)

Comment (nMoncho) [inline core-curriculum/schema/knowledge-graph.json]: Fixed, have the dictionary set to 'American English'

File: core-curriculum/schema/knowledge-graph.json (hunk 1 above)
Comment (nMoncho) [inline core-curriculum/schema/knowledge-graph.json]: Every now and then one of this escape me, thanks :)

**Pull request merged**